security/webauthn/src/test/java/MySqlWebAuthnIT.java

```diff
@@ -0,0 +1,32 @@
+import io.quarkus.test.bootstrap.MySqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MySqlWebAuthnIT extends AbstractWebAuthnTest {
+
+    private static final String MYSQL_USER = "quarkus_test";
+    private static final String MYSQL_PASSWORD = "quarkus_test";
+    private static final String MYSQL_DATABASE = "quarkus_test";
+    private static final int MYSQL_PORT = 3306;
+
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "ready for connections")
+    static MySqlService database = new MySqlService()
+            .withUser(MYSQL_USER)
+            .withPassword(MYSQL_PASSWORD)
+            .withDatabase(MYSQL_DATABASE);
+
+    @QuarkusApplication
+    static RestService app = new RestService().withProperties("mysql.properties")
+            .withProperty("quarkus.datasource.username", database::getUser)
```

Comment:
```suggestion
            .withProperty("quarkus.datasource.username", database.getUser())
```
It's static.

Reply:
I am using  method references, and think it is clear,  legible and works, but if you think this could cause some discrepancies because in the abstract DatabaseService those variables are not static so I will change it as you mention

Reply:
Value is set when initializing instance assigned to static field. The point here is not whether method reference is clear and it has nothing to do with discrepancies. The point is that value is always going to be same for `database":getUser` and `database.getUser()` and method reference will take additional space in metaspace, nah? I don't understand why to use it, that is all.

To be clear, these are unimportant things, I am pointing them out to provide you with information, I don't insist on changes.

Reply:
ok Michal, I really appreciate this kind of information, thank you.